tests/testAPI.sh

```diff
@@ -62,11 +75,25 @@ do
 		a)
 			AUTH_TOKEN="${OPTARG}"
 			;;
+		c)      
+			CLUSTER_TYPE="${OPTARG}"
+                        ;;
 		[?])
 			usage
 	esac
 done
 
+check_tool_existance "jq"
+
+if [[ $AUTH_TOKEN == "" ]]; then
+	if [[ -z $CLUSTER_TYPE ]]; then
+		echo "Error - Enter the Cluster Type !!"
```

Comment:
Here you have ask the user to either enter the AUTH_TOKEN or set the CLUSTER_TYPE.